.github/workflows/ci-tests.yaml

```diff
@@ -1,7 +1,12 @@
 name: Run tests
 
 on:
-  workflow_dispatch
+  workflow_dispatch: # allows manual triggering
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+  schedule: # run every day at midnight
+    -cron: '0 0 * * *'
```

Comment:
I'm not sure this will work correctly, but I can test it after merging... _always test the tests_...